README.md

```diff
@@ -24,13 +24,13 @@ then add your product to the list below.
 
 ## VPAT version history
 
-> Documents in this repository will conform to VPAT&reg; `Version 1.0`, `Version 2.2`, `Version 2.3`, `Version 2.4`, `Version 2.4 Rev` or the newer `Version 2.5`
+> Documents in this repository will conform to VPAT&reg; `Version 1.0`, `Version 2.2`, `Version 2.3`, `Version 2.4`, `Version 2.4 Rev`, `Version 2.5` or the newer `Version 2.5 Rev`
```

Comment:
I think it'd be better to remove versions 2.2 to 2.4 Rev (inclusive), because there's no documents using any of those - it's all either v1.0 or 2.5 Rev